src/hooks/usePatientData.tsx

```diff
@@ -23,10 +25,14 @@ const patientGenderMap = {
 
 export const usePatientData = (patientUuid) => {
   const { patient, isLoading: isLoadingPatient, error: patientError } = usePatient(patientUuid);
-  if (patient && !isLoadingPatient) {
+  const { isLoading: isLoadingPatientPrograms, error: patientProgramsError, activePatientEnrollment } = useActivePatientEnrollment(patientUuid);
+
+  if (patient && !isLoadingPatient && !isLoadingPatientPrograms) {
     // This is to support backward compatibility with the AMPATH JSON format
     patient['age'] = calculateAge(new Date(patient?.birthDate));
     patient['sex'] = patientGenderMap[patient.gender] ?? 'U';
+    patient['patientPrograms'] =  activePatientEnrollment;
```

Comment:
I am thinking of situations where `activePatientEnrollment()` resolves an empty dataset, should we be checking for that as well?